ext/droption/droption.h

```diff
@@ -680,23 +684,25 @@ inline bool
 droption_t<long>::convert_from_string(const std::string s)
 {
     errno = 0;
-    value_ = strtol(s.c_str(), NULL, 10);
+    value_ = strtol(s.c_str(), NULL, 0);
     return errno == 0;
 }
 template <>
 inline bool
 droption_t<long long>::convert_from_string(const std::string s)
 {
     errno = 0;
-    value_ = strtoll(s.c_str(), NULL, 10);
+    /* If we set 0 as the base, strtoll() will automatically identify the base like
```

Comment:
This file uses C++-style single-line comments (except doxygen). Maybe convert this and the above comment to use `//`.

Reply:
Done.